server/src/components/CodeExample.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+const code = `<span class="hljs-keyword">const</span> SmeeClient = <span class="hljs-built_in">require</span>(<span class="hljs-string">'smee-client'</span>)
+
+<span class="hljs-keyword">const</span> smee = <span class="hljs-keyword">new</span> SmeeClient({
+<span class="hljs-attr">  source</span>: <span class="hljs-string">'https://smee.io/sdfasdf'</span>,
```

Comment:
Thinking about the implementation in master, there is a dynamic piece to the code example:

https://github.com/probot/smee/blob/c03ef1fbe62aabd6cfbb9156347373fab7bf02a8/server/src/components/Blank.js#L10

Whereas this branch has a hardcode `source` URL. I don't have a preference either way, but just noticed that the behavior changed between master and this PR in case that's an issue.

Reply:
ooh good catch, this is definitely an unintentional change. We can un-hard-code this in the same way, by just using `window.location.href`.